.github/workflows/labels.yml

```
@@ -0,0 +1,128 @@
+            // First, handle label replacements/conversions
+            for (const label of labels) {
+              if (label.replaces && label.replaces.length > 0) {
+                for (const oldLabel of label.replaces) {
+                  try {
+                    // Check if the old label exists
+                    await github.rest.issues.getLabel({
+                      owner: owner,
+                      repo: repo,
+                      name: oldLabel
+                    });
+                    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)